src/client/Dockerfile

```diff
@@ -7,7 +7,7 @@ ARG REACT_APP_BUILD_VERSION
 
 COPY .npmrc .npmrc
 COPY ./package*.json ./
-RUN npm ci
+RUN npm install
```

Comment:
npm ci fails when trying to download the private package. I don't know how to fix this so I put back npm install at the moment...